engine/gui/src/gui.h

```diff
@@ -751,6 +751,12 @@ namespace dmGui
      */
     void RenderScene(HScene scene, const RenderSceneParams& params, void* context);
 
+    /**
+     * Force deletion of the dynamic textures of the scene
+     * @param scene Scene for which to remove the dynamic textures
+     */
+    void ForceRemoveDynamicTextures(HScene scene, DeleteTexture m_DeleteTexture);
```

Comment:
The description says "force deletion" while the function name is "ForceRemove". I think "Delete" is more appropriate than "Remove".

I also wonder about "Force". Can't this be omitted from the name? DeleteDynamicTextures() seems descriptive enough of what the function does.

Reply:
We already have `ClearTextures()`, so I think `ClearDynamicTextures` would be better

Reply:
Hmm, but we are deleting the textures are we not? ClearTextures() seems to only cancel animations and null the pointers/handles. I don't think ClearDynamicTextures() properly conveys that the textures are deleted.

Reply:
Now it's `DeleteDynamicTextures()`